relayer/chains/substrate/event_parser.go

```diff
@@ -289,6 +331,17 @@ func (ch *channelInfo) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 func (ch *channelInfo) parseAttrs(log *zap.Logger, attributes interface{}) {
 	attrs := attributes.(ibcEventQueryItem)
 
+	var err error
+
+	var height exported.Height
+	if height, err = parseHeight(attrs["height"]); err != nil {
+		log.Error("error parsing channel height: ",
```

Comment:
Logs should begin with a capital letter and not end in punctuation.

See https://github.com/cosmos/relayer/blob/main/docs/logging.md